html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <form action="index.html" method="GET">
+      <h1>Coding Form!</h1>
+      <div><label for="e-mail">Email</label></div>
+      <input type="email" id="e-mail">
+
+      <div>Why is coding cool?</div>
+      <select name="dropdown">
+        <option>Select an option</option>
+        <option>Because it's interesting</option>
+        <option>Because it's fun</option>
+        <option>Because it's powerful</option>
+      </select>
+
+      <div>Favorite Language(Choose one)</div>
+      <input type="radio" name="radio-button" value="HTML"/> HTML
+      <input type="radio" name="radio-button" value="CSS"/> CSS
+      <input type="radio" name="radio-button" value="JavaScript"/> JavaScript
+
+      <div><label for="checkbox">Would you like to sign up for more information?</label></div>
+      <input type="checkbox" name="checkbox" id="checkbox" value="Sign me up!" /> Sign me up!
+
+      <div>Comments</div>
```

Comment:
This should be a `label` element, with a `for` attribute corresponding to an `id` on the target input element.